### DISCUSSION TITLE
How to add permissions for other admins

### DISCUSSION THREAD

**Discussion by DruFin-debug** [svrooij/WinTuner#99] [Q&A]
How would you add permissions in the Tenant Enterprise Applications for Wintuner to be able to publish by other co-workers who didn't create the Enterprise Application?

**Comment by svrooij** [comment]
If you use interactive authentication (the Windows Hello integration), the application talks to Intune (graph api actually) in the context of the user. So apart from that user to be allowed to use that application, they would also need to be Intune Administrator (or whatever role gives access to managing applications in Intune).

So the question is: what authentication method are you using? And are the other users also Intune Administrator?

**Comment by DruFin-debug** [comment]
Currently using User Auth,
How do I add other users to use the application in Entra is the bigger question.  I have added the other users in the Users and Groups area on the Enterprise Application.

**Reply by svrooij** [reply]
The user itself has to be able to do it in the portal (a.k.a. Intune Administrator or higher)

It is using the account of the user.
